simplq/src/components/common/Button/Button.jsx

```diff
@@ -2,14 +2,14 @@ import React from 'react';
 import styles from '../../../styles/buttons.module.scss';
 
 const StandardButton = (props) => {
-  const { onClick, text } = props;
+  const { onClick } = props;
   return (
     <button
       type="submit"
       onClick={onClick}
-      className={props.className ? props.className : styles['standard-button']}
+      className={props.dark ? styles['standard-button-dark'] : styles['standard-button']}
```

Comment:
rather than allowing class override, which we don't use as of now, I've introduced a dark mode

Reply:
What do you think about creating a dark theme altogether for the site instead of assigning separate light and dark class styles for each component? A theme like what material UI uses. This is kind of what I'm talking about: https://medium.com/@dmitriy.borodiy/easy-color-theming-with-scss-bc38fd5734d1

Reply:
ok, so `dark` was maybe a bad name. We have two types of buttons in our design. One is the outlines one, like these:

![image](https://user-images.githubusercontent.com/11256376/89315146-2279c480-d698-11ea-8855-cd92dc626c5e.png)

And then there are filled ones:

![image](https://user-images.githubusercontent.com/11256376/89315302-4c32eb80-d698-11ea-84ac-4822910a4ff3.png)

This is different from light/dark theme. I'll rename from `dark` to `outlined`. Then there would be two css classes, `standard-button` and `standard-button-outlined`. Is there a better way to handle this?

Reply:
Yeah, best to rename. We could either use separate classes or create two different button components altogether for both of these, but two components for just the outline seems like overkill. So, let's just use one component with these two classes for now and change later if required.

Reply:
done